rotkehlchen/db/settings.py

```diff
@@ -106,6 +118,7 @@ class DBSettings(NamedTuple):
     pnl_csv_have_summary: bool = DEFAULT_PNL_CSV_HAVE_SUMMARY
     ssf_0graph_multiplier: int = DEFAULT_SSF_0GRAPH_MULTIPLIER
     last_data_migration: int = DEFAULT_LAST_DATA_MIGRATION
+    non_syncing_exchanges: Optional[List[ExchangeLocationID]] = None
```

Comment:
Maybe here you can define as default a `[]`?

Reply:
Yep, I've already added it in the new version

Reply:
Ehm ... @yabirgb and @nebolax you both passed the interview for python knowledge. Don't you remember the mutable default arguments question?

```python
from typing import List, NamedTuple


class Foo(NamedTuple):
    my_list: List[int] = []

foo1 = Foo()
foo1.my_list.append(42)

foo2 = Foo()
print(foo2.my_list)  # prints [42]
```

Reply:
Maybe I understood bad how the settings class is used but those settings are not supposed to be modified in this structure. Also we have other mutable list as default value there that should be updated https://github.com/rotki/rotki/blob/2394e0aacce4147c1ad3527864f4dc1234abc12a/rotkehlchen/db/settings.py#L41

Reply:
Objects of this class is not supposed to be modified indeed. But if it's modified BOOM! And it's easy to do by mistake.

So yeah we may want to address this. Let's think of what the best way to do so would be.

Reply:
Maybe we could make it None and perform a check each time we access this field?

Reply:
no no having an extra check each time we access is bad for performance. Probably can leave as is for now and see later.